packages/components/psammead-image-placeholder/src/index.test.jsx

```diff
@@ -19,4 +19,8 @@ describe('ImagePlaceholder', () => {
     'should render square images correctly',
     <ImagePlaceholder ratio={squareImageRatio} />,
   );
+  shouldMatchSnapshot(
```

Comment:
Should there be an updated snap to go along with this?

Reply:
There absolutely should! Was sat in my unstaged work. Veery nice spot.